debugging/book-library/script.js

```diff
@@ -25,79 +21,98 @@ const author = document.getElementById("author");
 const pages = document.getElementById("pages");
 const check = document.getElementById("check");
 
-//check the right input from forms and if its ok -> add the new book (object in array)
-//via Book function and start render function
+// Add event listener to the Submit button
+document.querySelector(".btn-primary").addEventListener("click", function (e) {
+  e.preventDefault();  // Prevent form submission
+  submit();  // Call the submit function
+});
+
+// Check if input is valid and then add the book
 function submit() {
+  // Validate inputs
   if (
-    title.value == null ||
-    title.value == "" ||
-    pages.value == null ||
-    pages.value == ""
+    title.value.trim() === "" ||  // Check for empty title
+    author.value.trim() === "" || // Check for empty author
+    pages.value.trim() === ""     // Check for empty pages
   ) {
     alert("Please fill all fields!");
     return false;
   } else {
-    let book = new Book(title.value, title.value, pages.value, check.checked);
-    library.push(book);
+    // Corrected to add author and title properly
+    let book = new Book(title.value, author.value, pages.value, check.checked);
+    myLibrary.push(book);  // Push to myLibrary, not library
     render();
+    resetForm(); // Reset the form after adding the book
   }
 }
 
+// Book constructor function
 function Book(title, author, pages, check) {
   this.title = title;
   this.author = author;
   this.pages = pages;
-  this.check = check;
+  this.read = check;  // Renamed 'check' to 'read' for clarity
```

Comment:
If you renamed `check` to `read` for clarity, what about `check` on line 22 & 116? Should it be also renamed?

Reply:
I've changed  all the checks to reads.